src/com/ebcitron/piggybank/Bank.java

```diff
@@ -0,0 +1,34 @@
+package com.ebcitron.piggybank;
+import java.util.*;
+import  java.text.DecimalFormat;
+
+
+public class Bank {
+
+
+    private ArrayList<Coin> coinArr = new ArrayList<Coin>();
+
+    DecimalFormat DF = new DecimalFormat("$###,###.00");
+
+    public Bank() {
+
+    }
+
+    public  void addCoin(Coin newCoin){
+        coinArr.add(newCoin);
+    }
+
+    public void countFunds() {
+
+        double arrValue = 0;
+        for (Coin coin : coinArr) {
+            System.out.println(coin.toString());
```

Comment:
You should fill out the toString() method in your coin, as right now it's only spitting out memory addresses